xmake/modules/package/manager/cmake/configurations.lua

```diff
@@ -28,6 +28,8 @@ function main()
         moduledirs     = {description = "Set the cmake modules directories."},
         presets        = {description = "Set the preset values, e.g. {Boost_USE_STATIC_LIB = true}"},
         envs           = {description = "Set the run environments of cmake, e.g. {CMAKE_PREFIX_PATH = \"xxx\"}"},
+        find_name      = {description = "Set the name used for ${find_name}_LIBS/_INCLUDE_DIRS in cmake"},
+        debug          = {description = "Set the CMAKE_BUILD_TYPE to Debug if set to true"},
```

Comment:
debug is builtin configuration, we need not define it. and you can get `opt.mode == debug` directly

Reply:
this flag should actually be called as `spread_debug`. Once set, it will find debug version of external library if xmake is under debug mode.

Do we need this flag, or we just set `CMAKE_BUILD_TYPE` to `Debug` automatically internally?

Reply:
I end up removing `debug` flags, since `CMKAE_BUILD_TYPE` is a global variable and will spread to all linked libraries. This is consistent with cmake behavior.

Reply:
you need pass to configs.debug

```
add_requires("cmake::ZLIB", {configs = {debug = true}})
```

```
{
  require_version = "latest",
  mode = "debug",   <------- debug
  configs = {
    debug = true,
    shared = false,
    pic = true
  },
  arch = "x86_64",
```